test/expect-tests/dune_pkg/dune_pkg_unit_tests.ml

```diff
@@ -228,88 +223,86 @@ let%expect_test "encode/decode round trip test for lockdir with complex deps" =
   let module Action = Dune_lang.Action in
   let module String_with_vars = Dune_lang.String_with_vars in
   lock_dir_encode_decode_round_trip_test ~lock_dir_path:"complex_lock_dir"
```

Comment:
The large diffs in this file are from whitespace changes due to a function being replaced by a value.